mapper/orm/pom.xml

```diff
@@ -71,69 +71,31 @@
-                        <artifactId>moditect-maven-plugin</artifactId>
-                        <executions>
-                            <execution>
-                                <id>add-module-infos</id>
-                                <!-- Disable the plugin, see above -->
-                                <phase>none</phase>
-                                <goals>
-                                    <goal>add-module-info</goal>
-                                </goals>
-                                <configuration>
-                                    <module>
-                                        <moduleInfo>
-                                            <name>${java.module.name}</name>
-                                            <exports>
-                                                !org.hibernate.search.*.impl;
-                                                !org.hibernate.search.*.impl.*;
-                                                *;
-                                            </exports>
-                                            <requires>
-                                                !net.bytebuddy;
-                                                !net.bytebuddy.*;
-                                                !javax.xml;
-                                                !javax.xml.*;
-                                                !javax.*;
-                                                *;
-                                            </requires>
-                                        </moduleInfo>
-                                    </module>
-                                </configuration>
-                            </execution>
-                        </executions>
-                    </plugin>
-                </plugins>
-            </build>
-        </profile>
-    </profiles>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.moditect</groupId>
+                <artifactId>moditect-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>add-module-infos</id>
+                        <configuration>
+                            <module>
+                                <moduleInfo>
+                                    <name>${java.module.name}</name>
+                                    <exports>
+                                        org.hibernate.search.*.impl to org.hibernate.search.util.common, org.hibernate.search.engine, org.hibernate.search.backend.elasticsearch, org.hibernate.search.backend.elasticsearch.aws, org.hibernate.search.mapper.pojo, org.hibernate.search.mapper.javabean, org.hibernate.search.mapper.orm, org.hibernate.search.batch.jsr352.core, org.jboss.logging;
+                                        *;
+                                    </exports>
+                                    <requires>
+                                        !cdi.api;
```

Comment:
Why do you remove this, though? Please add a comment here, at least.

I suspect this means Hibernate Search won't work with CDI when it's in the modulepath?

Reply:
Bump.

If there is no `cdi.api` module, why do you need to mention it here? Is it automatically detected? In that case, it must exist, right?

Maybe you would rather need some kind of optional dependency? Something like [`requires static`](https://nipafx.dev/java-modules-optional-dependencies/#optional-dependencies-with-requires-static)?